modules/spo-applying-profiles.adoc

```diff
@@ -0,0 +1,135 @@
+// Module included in the following assemblies:
+//
+// * security/security_profiles_operator/spo-seccomp.adoc
+// * security/security_profiles_operator/spo-selinux.adoc
+
+ifeval::["{context}" == "spo-seccomp"]
+:seccomp:
+:type: seccomp
+:kind: SeccompProfile
+endif::[]
+ifeval::["{context}" == "spo-selinux"]
+:selinux:
+:type: SELinux
+:kind: SelinuxProfile
+endif::[]
+
+:_content-type: PROCEDURE
+[id="spo-applying-profiles_{context}"]
+= Applying {type} profiles to a pod
+
+Create a Pod to apply one of the created profiles.
+
+.Procedure
+
+ifdef::seccomp[]
+. Create a Pod object that defines a `securityContext`:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: test-pod
+spec:
+  securityContext:
+    seccompProfile:
+      type: Localhost
+      localhostProfile: operator/my-namespace/profile1.json
+  containers:
+    - name: test-container
+      image: nginx
+----
+
+. View the profile path of the `seccompProfile.localhostProfile` attribute by running the following command:
++
+[source,terminal]
+----
+$ oc -n my-namespace get seccompprofile profile1 --output wide
+----
++
+.Example output
+[source,terminal]
+----
+NAME       STATUS   AGE   SECCOMPPROFILE.LOCALHOSTPROFILE
+profile1   Active   14s   operator/my-namespace/profile1.json
+----
+
+. Apply the profile to a Deployment by running the following command:
++
+[source,terminal]
+----
+$ oc --namespace my-namespace patch deployment myapp --patch '{"spec": {"template": {"spec": {"securityContext": {"seccompProfile": {"type": "localhost", "localhostProfile": "'$(oc --namespace my-namespace get seccompprofile profile1 --output=jsonpath='{.status.seccompProfile\.localhostProfile}')'}}}}}}'
+----
++
+.Example output
+[source,terminal]
+----
+deployment.apps/myapp patched
+----
+
+.Verification
+
+* Confirm the profile was applied correctly by running the following command:
++
+[source,terminal]
+----
+$ oc -n my-namespace get deployment myapp --output=jsonpath='{.spec.template.spec.securityContext}' | \
```

Comment:
The command lists the currently used securityContext for the deployment. `.spec.template` in a Deployment refers to how the pods managed by the deployment should look like, so in other words, "give me the security context of the pods that the deployment would create and manage".

JQ can be omitted, it's just used just as a formatter in this case. The only downside would be that the whole profile would be printed on a single line, something like:
```
{ "seccompProfile": { "localhostProfile": "operator/my-namespace/profile1.json", "type": "localhost" } }
```
Since we are setting a seccomp profile explicitly, if you want to make the output more compact, maybe the jsonpath could include the `seccompProfile`, too, e.g. --output=jsonpath='{.spec.template.spec.securityContext.seccompProfile}'